### PR TITLE
Add docs on pointing OTeL browser instrumentation to Highlight

### PR DESCRIPTION
**Pull request by ccschmitz**
## Summary

* Adds a page explaining how to point the OTeL browser instrumentation directly to Highlight's OTeL collector.
* Updates the client API docs to cover `startSpan` and `startManualSpan`.
* Updates the logic for finding the related session on the trace panel so that it works on `documentLoad` spans which don't have the session ID on the root level span (this is random - just noticed while going over a YouTube video w/ @jay-khatri).

## How did you test this change?

Click tested the docs to verify the new page looks good and ran a quick test in an E2E app to point traces to Highlight to ensure instructions were correct.

## Are there any deployment considerations?

N/A

## Does this work require review from our design team?

N/A

### CODE REVIEW

File: docs-content/getting-started/7_native-opentelemetry/5_browser.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* `service.name` (optional) - Specify the service name you want to assign on your traces. This makes it easier to filter to all traces from this instrumentation.
+
+You can set these attributes when creating your trace provider:
+
+```ts
+const provider = new WebTracerProvider({
+  resource: new Resource({
+    // TODO: Explain how to get the project and session IDs.
+    'highlight.project_id': 'your-project-id',
+    'highlight.session_id': 'your-session-id',
```
Comment (ccschmitz) [inline docs-content/getting-started/7_native-opentelemetry/5_browser.md:28]: @Vadman97 have any recommendations on how to tell people to access this? I didn't see anything that allowed you to access just the session ID, only the session URL.

Comment (ccschmitz) [reply docs-content/getting-started/7_native-opentelemetry/5_browser.md:27]: Ended up adding `sessionSecureID` to the object returned from `H.getSessionDetails()`. @Vadman97 let me know if you see any issues with that!

Comment (Vadman97) [reply docs-content/getting-started/7_native-opentelemetry/5_browser.md:27]: yeah that makes sense to me - the `H.init()` call also returns it but probably a simpler API to use the `getSessionDetails()` call

File: docs-content/sdk/client.md [hunk 2]
```diff
@@ -245,10 +245,91 @@ slug: client
   </div>
 </section>
 
+<section className="section">
+  <div className="left">
+    <h3>H.startSpan</h3>
```
Comment (Vadman97) [inline docs-content/sdk/client.md:250]: one thing i noticed with the `startSpan` and `startManualSpan` API is that it doesn't allow returning the value of the callback in some cases. should we make it clear that returning the return value of the `cb` is / is not an option?

**Review by Vadman97 — APPROVED** [review]
🚢

File: docs-content/sdk/client.md (hunk 2 above)
Comment (ccschmitz) [reply docs-content/sdk/client.md:250]: Good callout, @Vadman97! I ended up fixing that in https://github.com/highlight/highlight/pull/9335/commits/984435d30f5529a92c9dc2c694a86d59986c9ce0. Mind giving those changes a quick look? Will discard your previous review - thanks for looking this over!

File: sdk/firstload/src/index.tsx [hunk 3]
```diff
@@ -380,25 +381,25 @@ const H: HighlightPublicInterface = {
 	): any => {
 		const tracer = typeof getTracer === 'function' ? getTracer() : undefined
 		if (!tracer) {
+			console.log('::: no tracer')
```
Comment (Vadman97) [inline sdk/firstload/src/index.tsx:384]: remove

File: sdk/firstload/src/index.tsx [hunk 4]
```diff
@@ -380,25 +381,25 @@ const H: HighlightPublicInterface = {
...
-		const wrapCallback = async (
-			span: Span,
-			callback: (span: Span) => any,
-		) => {
+		const wrapCallback = (span: Span, callback: (span: Span) => any) => {
+			let result: any
 			try {
-				const result = await callback(span)
-				return result
+				result = callback(span)
```
Comment (Vadman97) [inline sdk/firstload/src/index.tsx:399]: if the callback is a promise, does that mean line 401 (`span.end()`) will happen before the promise is finished? seems like a regression in expected behavior

**Review by Vadman97 — APPROVED** [review]

File: docs-content/sdk/client.md (hunk 2 above)
Comment (Vadman97) [reply docs-content/sdk/client.md:250]: reviewed!

File: sdk/firstload/src/index.tsx (hunk 4 above)
Comment (ccschmitz) [reply sdk/firstload/src/index.tsx:399]: Ah, you're right! Thanks for catching that. I think there was also a bug with our types for this function which should have said it is returning a promise.

Comment (ccschmitz) [reply sdk/firstload/src/index.tsx]: Just addressed this and added a test for async callbacks in https://github.com/highlight/highlight/pull/9335/commits/3099b9fe042aa312e84dbf41b3e6c3ecfdf8e46f

**Review by Vadman97 — APPROVED** [review]
makes sense to me

**Pull request merged**